### DISCUSSION TITLE
HA problemi con fastweb

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2927] [HUB: Home Assistant]
Chiedo aiuto qui perché sembra il problema HA.
Oramai da diversi giorni sto riscontrando problemi con adsl fastweb.

Con un tempo variabile da 10minuti a 2ore si blocca la connessione internet ma il modem segnala la portante.

Dopo tanti interventi da parte di tecnici fastweb e varie prove interne il problema sembra presentarsi solo quando il Rasberry con HomeAssistant è collegato sia via lan che wifi.

Possibile che HomeAssistant blocchi la connessione fastweb?
Posso fare qualcosa?

Grazie

**Comment by snakuzzo** [comment]
No, HA non blocca alcuna connessione.
Poi se hai altro up&running sul rpi che satura la banda, puoi saperlo solo tu...ma il tema direi che è totalmente off-topic, per cui... discussione chiusa e, cortesemente, non ne aprire altre (visto che questa è la seconda), grazie.
